### PR TITLE
Update go-swagger

### PR DESCRIPTION
**Pull request by hasit**
### Description of change

- This update fixes adding descriptions to inner struct fields
- Removed deprecated fields

##### Checklist

- [ ] Tested in playground or other setup
- [ ] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [ ] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

### Summary by OpenAI

**Refactor:**
- Remove deprecated fields and fix descriptions for inner struct fields
- Update go-swagger package and configurations
- Rename `BatchPrerolupConfig` to `BatchPrerollupConfig`

**Documentation:**
- Update documentation and add/remove descriptions

**Test:**
- Comment out test cases with TODO message

> 🎉 Deprecated fields are gone, our code is now lean,
> Descriptions fixed, and swagger's updated sheen.
> Renamed configs, docs improved with care,
> A PR full of changes, a breath of fresh air! 🌟

### CODE REVIEW

File: cmd/aperture-agent/config/types.go [hunk 1]
```diff
@@ -58,9 +58,9 @@ type AgentOTelConfig struct {
 	otelconfig.CommonOTelConfig `json:",inline"`
 	// DisableKubernetesScraper disables metrics collection for Kubernetes resources.
 	DisableKubernetesScraper bool `json:"disable_kubernetes_scraper" default:"false"`
-	// BatchPrerollup configures batch_prerollup processor.
+	// BatchPrerollup configures batch pre rollup processor.
```
Comment (harjotgill) [inline cmd/aperture-agent/config/types.go:61]: To resolve these issues you can use code block. e.g. `batch_prerollup`. Though in this case it makes sense to provide a more descriptive comment.

File: pkg/policies/flowcontrol/selectors/control-point-id.go [hunk 2]
```diff
@@ -125,11 +124,6 @@ func TypedGlobalControlPointIDFromProto(protoCP *cmdv1.GlobalFlowControlPoint) T
...
 	ctrlPt := flowSelectorMsg.FlowMatcher.GetControlPoint()
 	service := flowSelectorMsg.ServiceSelector.GetService()
-	// map all to catch-all service for backward compatibility
-	// Deprecated: v1.5.0
-	if service == "all" {
-		service = consts.AnyService
-	}
 	return ControlPointID{
 		Service:      service,
 		ControlPoint: ctrlPt,
```
[the remark this answers is not in the document]
Comment (hdkshingala) [reply pkg/policies/flowcontrol/selectors/control-point-id.go:129]: This change was intentional and removes the deprecated fields so no need of backward compatibility

File: pkg/policies/controlplane/validator_test.go [hunk 3]
```diff
@@ -83,36 +83,37 @@ metadata:
...
-            extractor:
-              from: request.http.headers.user_type
+    flow_control:
+      flux_meters:
+        "service_latency":
+          flow_selector:
+            service_selector:
+              service: "service1-demo-app.demoapp.svc.cluster.local"
+            flow_matcher:
+              control_point: "ingress"
```
[the remark this answers is not in the document]
Comment (hdkshingala) [reply pkg/policies/controlplane/validator_test.go:93]: It is already aligned

File: pkg/policies/controlplane/validator_test.go [hunk 4]
```diff
@@ -342,30 +343,32 @@ metadata:
...
+              package user_from_cookie
+              cookies := split(input.attributes.request.http.headers.cookie, "; ")
+              user := user {
+                  cookie := cookies[_]
+                  startswith(cookie, "session=")
+                  session := substring(cookie, count("session="), -1)
+                  parts := split(session, ".")
+                  object := json.unmarshal(base64url.decode(parts[0]))
+                  user := object.user
+              }
```
[the remark this answers is not in the document]
Comment (hdkshingala) [reply pkg/policies/controlplane/validator_test.go:367]: It is already aligned

File: cmd/aperture-agent/config/types.go (hunk 1 above)
Comment (hdkshingala) [reply cmd/aperture-agent/config/types.go]: Changed the description

File: pkg/policies/flowcontrol/service/envoy/authz_test.go [hunk 5]
```diff
@@ -111,33 +111,24 @@ var service1FlowSelector = policylangv1.FlowSelector{
...
+			Module: `
+				package envoy.authz
+				destination := v {
+					v := input.attributes.destination.address.socketAddress.address
+				}
+				source := v {
+					v := input.attributes.source.address.socketAddress.address
+				}
+			`,
 		},
```
[the remark this answers is not in the document]
Comment (hdkshingala) [reply pkg/policies/flowcontrol/service/envoy/authz_test.go:132]: This change is intentional based on recent changes

**Pull request merged**